scripts/index.js

```diff
@@ -0,0 +1,133 @@
+const state = {
+    tempCount: 70
+};
+
+const increaseTemp = () => {
+    state.tempCount += 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
+};
+
+const decreaseTemp = () => {
+    state.tempCount -= 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
+};
+
+const blue = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `blue`;
+};
+const green = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `green`;
+};  
+const yellow = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `yellow`;
+};  
+const orange = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `orange`;
+}; 
+const red = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `red`;
+};
+
+const updateName = (event) => {
+    const cityName = document.querySelector("#sampleCity");
+    cityName.textContent = event.target.value;
+};
+
+const resetName = () => {
+    const cityName = document.querySelector("#sampleCity");
+    cityName.textContent = "Seattle"
+}
+
+const cold = () => {
+    const floorLevel = document.querySelector("#floor");
+    floorLevel.textContent = "🌲______⛄️🌲🌲🏂🏻🌲__🌲";
+};
+const cool = () => {
+    const floorLevel = document.querySelector("#floor");
+    floorLevel.textContent = "🧗🏾____🌵🌻_🐿_🌷__🌷🌷";
+};
+const regular = () => {
+    const floorLevel = document.querySelector("#floor");
+    floorLevel.textContent = "🌻🌻🌻__🌻🌻_🐿_🌷__🌷🌷";
+};
+const mild = () => {
+    const floorLevel = document.querySelector("#floor");
+    floorLevel.textContent = "💃🏿💃🏾💃🏽💃🏼💃🏻__🚴🏿_🚴🏾🚴🏽🚴🏼🚴🏻";
+};
+const hot = () => {
+    const floorLevel = document.querySelector("#floor");
+    floorLevel.textContent = "🧗🏾____🌵🌵🌵__🐍__🌵🌵";
+};
+
+const pickSky = (event) => {
```

Comment:
Similar to the ground, we could make a data structure that both captures the relationship between the displayed sky label and the sky string, and then use that data both to build the UI and process user events.

```js
const skySettings = [
    { label: 'snowy', sky: '🌬​🌨​☁️☁️❄️☁️🌨❄️🌨☁️🌨🌨☁️' },
    { label: 'rainy', sky: '⛈🌧☁️☁️🌧🌧☁️🌧 🌦 🌧☁️☁️🌧' },
    { label: 'cloudy', sky: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️' },
    { label: 'sunny', sky: '☁️   ☁️ ☁️  ☀️ ☁️   ☁️ ✈️ ' },
];
```

Since this list is ordered, we could iterate over it to build the sky select options, and either set the sky string to be the option value directly, or still use the label to look into this list to get the sky string (or build a dictionary using the label as the key and the sky string as the value if we're _really_ worried about the length of the list).